### ISSUE TITLE
enumitem and beamer

### ISSUE THREAD

**Issue by zmberber** [cionx/writing-latex#3]
if i understand it correctly, enumitem and beamer don't work well together.  maybe you could mention that...

**Comment by cionx** [comment]
I guess it would make sense to add a general remark that `beamer` redefined many things in the background, and therefore easily leads to unexpected problems.

**Comment by zmberber** [comment]
good idea!  but i do think it is important to mention the specific quirks that are known, not just a general "stuff may break".  sometimes it is really hard to find the cause of an issue like that, and finding specifics like that can be helpful.
